frontend/src/Student/pages/Dashboard.js

```diff
@@ -8,7 +8,6 @@ const Dashboard = () => (
     <Row>
       <Col>
         <Shows />
-        {/* TODO: Show all open shows w/ link to any submissions a student made to them */}
```

Comment:
@JohnPMurray I actually don't know why this is here. Was this your addition?

Reply:
Wasn't me, should be good to delete